package.json

```diff
@@ -44,7 +44,9 @@
       "plugin:vue/essential",
       "@vue/prettier"
     ],
-    "rules": {},
+    "rules": {
```

Comment:
generally, for something like Regex instead of turning the rule off entirely we should use `// eslint-disable-next-line`